.github/workflows/android.yml

```diff
@@ -83,6 +83,27 @@ jobs:
         run: |
           build_scripts/android/install_prereqs.sh
 
+      - name: Add strings.exe to PATH (Windows only)
+        if: startsWith(matrix.os, 'windows')
+        shell: bash
+        run: |
+          # strings.exe was installed in repo root by install_prereqs.sh.
+          # There is another conflicting strings executable on Github runners.
+          # "C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\bin\strings.exe"
+          # Make sure to prepend the one in our repo root to the path.
+          echo "${{ github.workspace }}" >> $GITHUB_PATH
+
+      - name: Run strings.exe once to accept license(Windows only)
+        if: startsWith(matrix.os, 'windows')
+        continue-on-error: true
+        shell: bash
+        run: |
+          # This invocation is just to accept the license. Otherwise the command
+          # fails during build process. Also, running a command with just this
+          # flag makes the process exit with code 127 and stops the workflow.
+          # We ignore the error and continue with the rest of the steps.
+          strings -accepteula
+
```

Comment:
Nit: Our workflow logs are getting pretty verbose.  I think it would be better if these were in the same step. WDYT?

Reply:
ProTip: in bash, you can ignore the error by just running as: strings -accepteula || true

Reply:
Unfortunately they can't be in the same step. Github's PATH update takes effect in the following steps/actions. There is no other Windows specific  tasks in this workflow that we can place this or the update in.